File: C#/PalindromNumber.cs
using System;

class Program
{
    static void Main()
    {
        int num, reversedNum = 0, remainder, originalNum;

        // Input a number
        Console.Write("Input a number: ");
        num = int.Parse(Console.ReadLine());

        originalNum = num;

        // Reverse the number
        while (num != 0)
        {
            remainder = num % 10;
            reversedNum = reversedNum * 10 + remainder;
            num /= 10;
        }

        // Check if the original number is equal to the reversed number
        if (originalNum == reversedNum)
        {
            Console.WriteLine($"{originalNum} is a palindrome number.");
        }
        else
        {
            Console.WriteLine($"{originalNum} is not a palindrome number.");
        }
    }
}

File: C/PalindromNumber.c
#include <stdio.h>

int main() {
    int num, reversedNum = 0, remainder, originalNum;

    // Input a number
    printf("Input a number: ");
    scanf("%d", &num);

    originalNum = num;

    // Reverse the number
    while (num != 0) {
        remainder = num % 10;
        reversedNum = reversedNum * 10 + remainder;
        num /= 10;
    }

    // Check if the original number is equal to the reversed number
    if (originalNum == reversedNum) {
        printf("%d is a palindrome number.\n", originalNum);
    } else {
        printf("%d is not a palindrome number.\n", originalNum);
    }

    return 0;
}
